src/views/studio/MyVideosView/MyVideosView.tsx

```diff
@@ -297,6 +301,14 @@ export const MyVideosView = () => {
               />
             )}
             <Grid maxColumns={null} onResize={handleOnResizeGrid}>
+              {currentPage === 0 && (
+                <NewVideoTile to={absoluteRoutes.studio.editVideo()}>
```

Comment:
This kind of breaks the pagination on this page - now the first page will have items for 4 rows + 1 item in 5th row